Doc/library/tarfile.rst

```diff
@@ -100,7 +100,10 @@ Some facts and figures:
 
    For modes ``'w:gz'``, ``'r:gz'``, ``'w:bz2'``, ``'r:bz2'``, ``'x:gz'``,
    ``'x:bz2'``, :func:`tarfile.open` accepts the keyword argument
-   *compresslevel* (default ``9``) to specify the compression level of the file.
+   *compresslevel* to specify the compression level of the file.
+   This defaults to ``9`` for gzip and bz2 compression and to
+   :class:`PRESET_DEFAULT <lzma.LZMACompressor>` for lzma compression.
```

Comment:
An alternative would be to continue to allow `tarfile.open(..., preset=9)` or whatever (which was never documented) and note that the different modes have different keywords? Or to check for both?